com.unity.render-pipelines.high-definition/Documentation~/whats-new-13.md

```diff
@@ -4,4 +4,15 @@ This page contains an overview of new features, improvements, and issues resolve
 
 ## Added
 
+### Material Runtime API
+
+To enable or disable most HDRP shader features on a material, changes need to be done on the keyword state and sometimes on one or more properties. This is done automatically when editing values though the inspector, but from HDRP 13.0, new APIs are also available in order to run the validation steps from script, both in the editor and in standalone builds.
```

Comment:
From this HDRP version, you can use new APIs to run shader validation steps from script at runtime, both in the editor and in standalone builds. You can use this to change the keyword state or one or more properties in order to enable or disable HDRP shader features on a Material.